redfish-core/lib/ethernet.hpp

```diff
@@ -1886,6 +1890,11 @@ inline void requestEthernetInterfacesRoutes(App& app)
 
                 if (dhcpv6)
                 {
+                    messages::actionNotSupported(
+                        asyncResp->res,
+                        "Setting of any ipv6 dhcp property on the interface");
```

Comment:
Please rephrase to : Setting DHCPv6 parameters on the interface

Reply:
Done